packages/toolpad-studio/src/canvas/index.tsx

```diff
@@ -152,6 +151,15 @@ export default function AppCanvas({ basename, state: initialState }: AppCanvasPr
       return { nodes };
     });
 
+    setCommandHandler(bridge.canvasCommands, 'scrollComponent', (nodeId) => {
+      if (!nodeId) {
+        return;
+      }
+      invariant(appRootRef.current, 'App root not found');
```

Comment:
Maybe you can also just check if `appRootRef.current` is truthy  instead of using `invariant` and just return if not?

Reply:
Just copy pasted that line for the one of the command written above, should I remove the `invariant`?

Reply:
Yeah, not a big deal but I think it makes more sense to just not scroll if this ref isn't there yet.